common/src/main/resources/assets/rrls/lang/zh_cn.json

```diff
@@ -0,0 +1,12 @@
+{
+  "text.autoconfig.rrls.title": "移除重新加载屏幕(RemoveReloadingScreen)",
```

Comment:
Leave either the translated text or the English

Reply:
The original text is used to distinguish mod (on the mod website and when searching for mod in English ), the translated text is for explanatory purposes